### ISSUE TITLE
Is it possible to have multiple wikis?

### ISSUE THREAD

**Issue by sgama** [google/docsy#216]
I was wondering if there was a way there could be multiple sections within a website using Docsy with wikis.

- Documentation
- Design
... etc

I know Documentation and Design could go in the same page, but I'm generalizing for the simplicity of the question.

**Comment by frodriguezsmartclip** [comment]
@sgama I think yes, you can have a unique domain, with differents paths to each wiki: documentation design...

Here in their example ([documentation docsy](https://example.docsy.dev/docs/)), you can have multiple folders to achieve this.

**Comment by sgama** [comment]
I may have not been very clear with my question, but I figured out how to have two sections with a docs layout.

I just copied the files from: [layouts/docs/](https://github.com/google/docsy/tree/2815ec5c5a41a561dc6e84cea2f4c9c532f1e401/layouts/docs) and placed them into a new folder named `content/en/design`.

I also added an menu item in the `config.toml`:
```
[[menu.main]]
    name = "Design"
    weight = 20
    url = "/design/"
```

Now I have two folders with the `docs` layout. I guess that's all I needed to get the layout.
